spinalcordtoolbox/utils/shell.py

```diff
@@ -87,9 +89,15 @@ def display_viewer_syntax(files, verbose, colormaps=[], minmax=[], opacities=[],
     return cmd_strings
 
 
-def _construct_fslview_syntax(viewer, files, colormaps, minmax, opacities, mode):
+def _construct_fslview_syntax(viewer, files, colormaps, im_types, minmax, opacities, mode):
     dict_fslview = {'gray': 'Greyscale', 'red-yellow': 'Red-Yellow', 'blue-lightblue': 'Blue-Lightblue', 'red': 'Red',
                     'green': 'Green', 'random': 'Random-Rainbow', 'hsv': 'hsv', 'subcortical': 'MGH-Subcortical'}
+    imtypes_colormap = {
+    'anat':    {'fsleyes': 'greyscale' ,  'fslview': 'Greyscale'},
+    'seg':     {'fsleyes': 'red',         'fslview': 'Red'},
+    'softseg': {'fsleyes': 'red-yellow',  'fslview': 'Red-Yellow'},
+    'labels':  {'fsleyes': 'subcortical', 'fslview': 'MGH-Subcortical'},
+    }
```

Comment:
It looks like your changes have a few "linting issues" (which means that some of the changed lines don't follow Python style guidelines).

Our automated check provides some feedback. You can find this feedback in the "[Details](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/4148079339/jobs/7175703504)" link for the "Check code quality" automated check:

![image](https://user-images.githubusercontent.com/16181459/218209177-524d9d01-a90a-4a50-9712-2adfcbc652a0.png)

```
spinalcordtoolbox/utils/shell.py:96:5: E122 continuation line missing indentation or outdented
spinalcordtoolbox/utils/shell.py:96:39: E203 whitespace before ','
spinalcordtoolbox/utils/shell.py:97:5: E122 continuation line missing indentation or outdented
spinalcordtoolbox/utils/shell.py:98:5: E122 continuation line missing indentation or outdented
spinalcordtoolbox/utils/shell.py:99:5: E122 continuation line missing indentation or outdented
spinalcordtoolbox/utils/shell.py:129:5: E122 continuation line missing indentation or outdented
spinalcordtoolbox/utils/shell.py:129:39: E203 whitespace before ','
spinalcordtoolbox/utils/shell.py:130:5: E122 continuation line missing indentation or outdented
spinalcordtoolbox/utils/shell.py:131:5: E122 continuation line missing indentation or outdented
spinalcordtoolbox/utils/shell.py:132:5: E122 continuation line missing indentation or outdented
```

For each of these warnings, you can google the code (e.g. "E122 python") and it will give you a suggestion for how to improve your code: https://www.flake8rules.com/rules/E122.html

Could you please address each of these linting warnings, so that the check passes? :smile:

Reply:
> It looks like your changes have a few "linting issues" (which means that some of the changed lines don't follow Python style guidelines).
> 
> Our automated check provides some feedback. You can find this feedback in the "[Details](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/4148079339/jobs/7175703504)" link for the "Check code quality" automated check:
> 
> ![image](https://user-images.githubusercontent.com/16181459/218209177-524d9d01-a90a-4a50-9712-2adfcbc652a0.png)
> 
> ```
> spinalcordtoolbox/utils/shell.py:96:5: E122 continuation line missing indentation or outdented
> spinalcordtoolbox/utils/shell.py:96:39: E203 whitespace before ','
> spinalcordtoolbox/utils/shell.py:97:5: E122 continuation line missing indentation or outdented
> spinalcordtoolbox/utils/shell.py:98:5: E122 continuation line missing indentation or outdented
> spinalcordtoolbox/utils/shell.py:99:5: E122 continuation line missing indentation or outdented
> spinalcordtoolbox/utils/shell.py:129:5: E122 continuation line missing indentation or outdented
> spinalcordtoolbox/utils/shell.py:129:39: E203 whitespace before ','
> spinalcordtoolbox/utils/shell.py:130:5: E122 continuation line missing indentation or outdented
> spinalcordtoolbox/utils/shell.py:131:5: E122 continuation line missing indentation or outdented
> spinalcordtoolbox/utils/shell.py:132:5: E122 continuation line missing indentation or outdented
> ```
> 
> For each of these warnings, you can google the code (e.g. "E122 python") and it will give you a suggestion for how to improve your code: https://www.flake8rules.com/rules/E122.html
> 
> Could you please address each of these linting warnings, so that the check passes? 🙂

I will do that ! Thanks for the explanation 😁